extensions/mongodb/src/test/resources/runDocker.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
I think this file can be excluded. Or if you think it makes sense to keep it with the tests, then please add a reasoning as a comment in the file.

Reply:
Removed whole file